docs/troubleshooting.md

```diff
@@ -0,0 +1,48 @@
+# Issues we met
+
+This document collects issues that we had with sssd-test-suite and their
+solution.
+
+## Ad machine sudden shutdown
+Probably license expired. Run
+
+    ./sssd-test-suite up -s ad ad-child && ./sssd-test-suite provision rearm
+
+
+## TASK [host : Install required vagrant plugins] failed due to vagrant dependencies
+
+remove ~/.vagrant.d
+uninstall vagrant-sshfs & vagrant-libvirt rpms and rerun host provisioning
+
+    rm -rf ~/.vagrant.d
+    sudo dnf remove vagrant-sshfs vagrant-libvirt
+    ./sssd-test-suite provision host -p "$POOL_DIR"
+
+
+## Various errors on fc28
+sssd-test-suite needs new ansibles, please upgrade to fc30.
+
+
+## error while downloading ad vagrant boxes during ./sssd-test-suite up -s
+
+An error occurred while downloading the remote file. The error
+message, if any, is reproduced below. Please fix this error and try
+again.
+
+SSL certificate problem: unable to get local issuer certificate
+
+Solution is to download ca chain for that website, store it
+into /etc/pki/ca-trust/source/anchors and run
+
+    update-ca-trust extract
```

Comment:
This applies only to internal vagrant boxes, please remove it. I'll update it internally.